sympy/utilities/lambdify.py

```diff
@@ -88,22 +90,74 @@
     "Matrix": "array",
     "MutableDenseMatrix": "array",
     "ImmutableMatrix": "array",
+    "ImmutableDenseMatrix": "array"
+
+}
+
+# most of the functions of scipy is similar to numpy. Checked each functions.
+SCIPY_TRANSLATIONS = {
+    "acos": "arccos",
+    "acosh": "arccosh",
+    "arg": "angle",
+    "asin": "arcsin",
+    "asinh": "arcsinh",
+    "atan": "arctan",
+    "atan2": "arctan2",
+    "atanh": "arctanh",
+    "ceiling": "ceil",
+    "E": "e",
+    "im": "imag",
+    "ln": "log",
+    "Max": "amax",
+    "Min": "amin",
+    "Mod": "mod",
+    "oo": "inf",
+    "re": "real",
+    "SparseMatrix": "array",
+    "ImmutableSparseMatrix": "array",
+    "Matrix": "array",
+    "MutableDenseMatrix": "array",
+    "ImmutableMatrix": "array",
     "ImmutableDenseMatrix": "array",
+    "elliptic_k": "ellipk",
+    "elliptic_f": "ellipeinc",
+    "besselj": "jn",
+    "bessely": "yn",
+    "besseli": "iv",
+    "besselk": "kn",
+    "loggamma": "gammaln",
+    "digamma": "psi",
+    "hyper": "hyp1f1",
 }
 
-NUMEXPR_TRANSLATIONS = {}
+NUMEXPR_TRANSLATIONS = {
```

Comment:
Oh! actually I wastrying some examples which used both scipy,numexpr and numpy,numexpr  , but I was getting some errors that time I thought mapping should be there for numexpr also.I will remove them and make different PR for this.